### DISCUSSION TITLE
Are there plans to include Excel formulas?

### DISCUSSION THREAD

**Discussion by NB-123** [silevis/reactgrid#226] [Q&A]
Hi Team,

This seems to be a really cool project. 

Are there any plans to include built-in Excel functions as well? 

For example, if I typed "=SUM(...)" in the cell, would it replicate Excel behavior at some point?

As an example, here is a paid product doing something like this: https://www.syncfusion.com/react-components/react-spreadsheet

Thank you:)

**Comment by Michota** [comment]
The v4 is currently is already in semi-maintance mode, which means we won't add any new features that are a major change, and formulas are major change. The v5 will be less opinionated, so it will most likely won't have a built-in formulas feature, though it may be able to handle it if we will manage to create some sort of API for extensions like this one.
